cookbook_se/recipes/native_typing/pyspark_pi.py

```diff
@@ -1,23 +1,20 @@
-import datetime
 import random
+import datetime
 from operator import add
-
 from flytekit import task, workflow
 from flytekit.taskplugins.spark import Spark
 
 
-@task(
-    task_config=Spark(
-        spark_conf={
-            "spark.driver.memory": "1000M",
-            "spark.executor.memory": "1000M",
-            "spark.executor.cores": "1",
-            "spark.executor.instances": "2",
-            "spark.driver.cores": "1",
-        }
-    ),
-    cache_version="1",
-)
+@task(task_config=Spark(
+    spark_conf={
+        'spark.driver.memory': "1000M",
+        'spark.executor.memory': "1000M",
+        'spark.executor.cores': '1',
+        'spark.executor.instances': '2',
+        'spark.driver.cores': '1',
+    }),
+    cache_version='1',
+    container_image='{{.image.default.fqn}}:spark-{{.image.default.version}}')
```

Comment:
so if you do this, you don't need to have a config file?

Reply:
nope, config file is needed for new image names, which you can directly hard code too!